Feature/louis

Louis02 opened the pull request.

Louis02 committed:
thread in main

Louis02 committed:
check

Louis02 committed:
Finished First Part

Louis02 committed:
# WARNING: head commit changed in the meantime

update

DaveDLeague committed:
Merge branch 'development' into feature/Louis

ryanoh0303 commented:
Nice Job! 

You coded discord bot that randomly assigns a random integer value and prints a line corresponding to the integer. It would have been better if you made it more complicated, but well done.

Commit:
working on logic of equation solver

Commit:
finished the equations

Commit:
added modulo and hold is adding parentheses

Louis02 committed:
done

Commit:
check for random spaces when parentheses are used because they don't
show up in the normal equations
need to sort that out then remove the equation within the parentheses
that is already solved

Commit:
need to work on remoing the leftovers of hold in the solveParenth method
